### PR TITLE
Minimal Debian system updates

### PR DESCRIPTION
**Pull request by cgzones**

### CODE REVIEW

File: policy/modules/system/init.if [hunk 1]
```diff
@@ -3752,6 +3752,26 @@ interface(`init_reload_all_units',`
...
+## </param>
+#
+interface(`init_getattr_all_unit_files',`
+	gen_require(`
+		attribute systemdunit;
+	')
+
+	list_dirs_pattern($1, systemdunit, systemdunit)
+	getattr_files_pattern($1, systemdunit, systemdunit)
+	getattr_lnk_files_pattern($1, systemdunit, systemdunit)
```
Comment (pebenito) [inline policy/modules/system/init.if:3772]: It seems like the symlink access here should actuallybe a read instead..

File: policy/modules/system/sysnetwork.if [hunk 2]
```diff
@@ -489,6 +489,7 @@ interface(`sysnet_create_config',`
 	')
 
 	files_search_etc($1)
+	allow $1 net_conf_t:dir create_dir_perms;
```
Comment (pebenito) [inline policy/modules/system/sysnetwork.if:492]: Perhaps this should be `{ create_dir_perms add_entry_dir_perms }`.

**Review by pebenito — COMMENTED** [review]
I think this could be merged as-is but have a couple thoughts.

File: policy/modules/system/init.if (hunk 1 above)
Comment (cgzones) [reply policy/modules/system/init.if]: done

File: policy/modules/system/selinuxutil.te [hunk 3]
```diff
@@ -249,6 +249,7 @@ read_lnk_files_pattern(newrole_t, default_context_t, default_context_t)
 
 kernel_read_system_state(newrole_t)
 kernel_read_kernel_sysctls(newrole_t)
+kernel_getattr_proc(newrole_t)
```
Comment (cgzones) [inline policy/modules/system/selinuxutil.te:252]: or should this be dontaudit'ed?

Comment (pebenito) [reply policy/modules/system/selinuxutil.te:252]: I go back and forth on this. Strictly speaking, dontaudit it, if it isn't required.

**Pull request merged**